<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="bbox_x" default="1.0"/>
	<arg name="bbox_y" default="1.0"/>
	<arg name="bbox_z" default="1.5"/>

	<node name="collision_detector" pkg="mader" type="collision_detector.py" output="screen">
		<param name="bbox_x" value="$(arg bbox_x)"/>
		<param name="bbox_y" value="$(arg bbox_y)"/>
		<param name="bbox_z" value="$(arg bbox_z)"/>
	</node>

</launch>



